### ISSUE TITLE
Mask Predict

### ISSUE THREAD

**Issue by DannyChen1994** [youngnuaa/iShapeInst#1]
Hello, I have installed the environment and tested **try_test_net.py** code can be run. I just started out and don't have a deep understanding of code. I haven't seen any code about mask visualization. May I ask if using the forward function in **try_demo_img. py**? Can you provide examples for guidance?

**Comment by youngnuaa** [comment]
the code specification is not too good. try_demo_img.py is used to show the result of model prediction, try_test_net.py is used to evaluate the map of the model, and train_net.py is used to train the model.

**Comment by DannyChen1994** [comment]
Thank you for your guidance. I have run through the previous code and successfully trained my dataset. But I found a problem when I evaluated the "model_final.pth" trained on my power-line synthesis dataset on the author's wire dataset. May I ask what may have caused this？Is it caused by not detecting the target？
![image](https://user-images.githubusercontent.com/46364265/231640505-05a273d3-2ea5-45f3-ab78-b08c0f78192d.png)

**Comment by youngnuaa** [comment]
I suggest print "cate_scores" to show the result.

**Comment by DannyChen1994** [comment]
Yes, I found that for graphs that have not been split into instances, the code will report an error, and 'cate_cores' is empty data. In addition, I found that when the input image is not square, such as the shape at 16:9, there will be a mismatch between the mask and the actual object, as shown in the following figure.
![wrong](https://user-images.githubusercontent.com/46364265/232673867-d2ba3151-1e1d-47b6-84d1-ca001f95c833.jpg)
When I modify the 'Base SpareInst. yaml' in the 'configs' folder (MAX_SIZE_TEST: 864). There is a mismatch in the mask in the high direction. May I ask what the reason is and how can we solve it?
![image](https://user-images.githubusercontent.com/46364265/232673774-6484f0d8-64cd-48be-a882-f859e2003924.png)

**Comment by DannyChen1994** [comment]
I made the following modification 
**MIN_SIZE_TEST: 1024
MAX_SIZE_TEST: 1024**
The wiping film is normal, and the problem described above is solved. Thank you！
